src/DEA/utils/DEA.R

```diff
@@ -0,0 +1,73 @@
+DEA <- function(symbols, WT_mean, KO_mean, other_IDs) {
```

Comment:
Yeah, especially since you have an `allDEA.R` function and a `DEA.R` function, it would be helpful to differentiate between the two. This can be done with a docstring

Reply:
the allDEA.R is not a function, its the script I used to utilize the function, you just have to have the function there to run it before using it (its different than in matlab where the function just needs to be in the same folder) but i will clarify this.